indico/htdocs/js/indico/jquery/itempicker.js

```diff
@@ -0,0 +1,125 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function($) {
+    $.widget('indico.itempicker', {
+
+        options: {
+            onSelect: null, // function called every time an item gets selected from the list
+            onActionButtonClicked: null,
+            createItem: null, // function returning a jQuery object representing each item in the dropdown
+            items: [], // all the options that should be included in the dropdown
+            containerClasses: '' // extra classes to be applied to the container
+        },
+
+        _create: function() {
+            var self = this;
+            var element = self.element;
+            var items = self.options.items.length ? self.options.items : (element.data('items') || []);
```

Comment:
then you can do `var items = self.options.items || element.data('items') || [];` here